3ds Max/Max2Babylon/Exporter/MaxExportParameters.cs

```diff
@@ -3,13 +3,23 @@
 
 namespace Max2Babylon
 {
+    public enum BakeAnimationType
+    {
+        DoNotBakeAnimation,
+        BakeAllAnimations,
+        BakeSelective
+    }
+
+
     public class MaxExportParameters : ExportParameters
     {
         public Autodesk.Max.IINode exportNode;
         public List<Autodesk.Max.IILayer> exportLayers;
         public bool usePreExportProcess = false;
+        public bool keepChangement = false;
```

Comment:
is keepChangement "Apply preprocessing effects to scene permanently"?

Reply:
yes I think is useful when many preprocess are applied and a user wants actually see what BabylonExporter is doing to his scene.
But this is not permanent as the scene is not saved and just reloading the scene manually a user can get back hi scene
Should I rename it?

Reply:
Makes sense, can we name this to "ApplyPreprocessToScene" or something similar? I just want to make sure that this option is clear to the user.